ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -52,8 +52,11 @@ def bufr2ioda(current_cycle, RUN, DMPDIR, config_template_dir, COM_OBS):
     }
 
     # copy necessary fix files to runtime directory
-    shutil.copy(os.path.join(config_template_dir, "atms_beamwidth.txt"),
-                os.path.join(DATA, "atms_beamwidth.txt"))
+    fix_files = ["atms_beamwidth.txt",
+                 "amsua_metop-c_v2.ACCoeff.nc"
```

Comment:
need to find a place for this .nc file...

Reply:
put a note in the code